datamade/la-metro-dashboard
# Replace Travis CI with GitHub Actions

## Description

See title.

## Testing instructions

- Confirm that a build successfully runs. Once merged, confirm that the app is deployed to staging. Finally, push to a `deploy` branch and confirm the app is deployed to production.
